libraries/ESP8266LLMNR/ESP8266LLMNR.cpp

```diff
@@ -242,8 +245,8 @@ void LLMNRResponder::_process_packet() {
 
     // Header
     uint8_t header[] = {
-        id >> 8, id & 0xff, // ID
-        FLAGS_QR >> 8, 0, // FLAGS
+        (uint8_t)(id >> 8), (uint8_t)(id & 0xff), // ID
```

Comment:
since there's a cast to uint8_t now, masking the low byte with `& 0xff` is not needed.